index.html

```diff
@@ -1955,13 +1955,29 @@
 <div class="practice">
 <p>
   <span class="practicelab" id="principle-consent-withdraw">
-    It should be as easy for a user to check what consent they have given,
+  It should be as easy for a [=person=] to check what consent they have given,
 	to withdraw consent, or to opt out or object,
 	as to give consent.
   </span>
 </p>
 </div>
 
+A [=person=] providing consent does not imply consent from other [=people=] whose data may be
+included or implicated.
+
+<div class="practice">
+  <p>
+    <span class="practicelab" id="principle-consent-otherpeople">
+      [=Actors=] should provide accessible functionality to access, correct and remove data about
+      [=people=] to those [=people=] when that data may have been provided by someone else.
```

Comment:
```suggestion
      [=people=] to those [=people=] when that data has been provided by someone else.
```

If it wasn't, this is just general data rights.